Library/Formula/nave.rb

```diff
@@ -0,0 +1,13 @@
+class Nave < Formula
+  homepage "https://github.com/isaacs/nave"
+  url "https://github.com/isaacs/nave/archive/v0.4.5.tar.gz"
+  sha1 "190fdab025882a9a50d85c2509b04f269f97dbf7"
+
+  def install
+    bin.install "nave.sh" => "nave"
+  end
+
+  test do
+    assert_match /Usage: nave/, shell_output("#{bin}/nave help")
```

Comment:
I upgrade to run `ls-remote` instead, which will at least establish functioning of the resultant formula beyond the basics of the shell.

I'm reticent to go further into testing as it quickly gets into testing Node.js builds themselves or testing network connectivity of the binary distribution web servers. I think `ls-remote` satisfies the "more than trivial" desire, but also avoids "testing something beyond the nave package itself" pitfall.

Let me know if you agree.